sdk/communication/AzureCommunicationChat/Source/Generated/Chat.swift

```diff
@@ -29,16 +29,16 @@ public final class Chat {
     ///    - options: A list of options for the operation
     ///    - completionHandler: A completion handler that receives a status code on
     ///     success.
-    public func create(
-        chatThread: CreateChatThreadRequest,
+    internal func create(
```

Comment:
Why is the `Chat` class public if all its methods are `internal`?  I don't understand what you are supposed to do with this.